libs/langchain/langchain/llms/ai21.py

```diff
@@ -141,9 +143,10 @@ def _call(
             else:
                 base_url = "https://api.ai21.com/studio/v1"
         params = {**self._default_params, **kwargs}
+        self.ai21_api_key = cast(SecretStr, self.ai21_api_key)
```

Comment:
Cast probably won't be needed if the attribute is no longer an Optional

Reply:
Yes, used the cast to avoid type checking, I was encountering a mypy error without it.